### DISCUSSION TITLE
I need to run the playframefork 2.5.7 project located in a private GitHub repository on the digitalocean server using "apps". Shows an error

### DISCUSSION THREAD

**Discussion by totupn** [playframework/playframework#12332] [Q&A]
It is necessary to connect the playframework 2.5.7 project from the github private repository to DigitalOcean with loading the PostgreSQL and java 8 database in order to start development with automatic compilation of sbt every time a change is made. When connecting to apps, the first step shows an error - the component was not detected. Although everything works locally. How to upload a project to the server step by step?

Please tell us!

![apps](https://github.com/playframework/playframework/assets/155981185/1b3462b4-10d8-4237-bdd4-53ace32eab22)

![no component](https://github.com/playframework/playframework/assets/155981185/b34a82f4-94cb-4d07-a00d-c73fa2edccbf)

**Comment by mkurz** [comment]
Not sure how this digital ocean app thing works. Is Play Framework actually supported by Digital Ocean?

**Reply by totupn** [reply]
I’m also interested in this, I couldn’t find a word about Play Framework in the Digital Ocean documentation. 
Maybe it’s possible to somehow install this Play without using “apps”, maybe someone knows..?
